src/Http/Requests/UserRegisterRequest.php

```diff
@@ -28,7 +28,7 @@ public function authorize(): bool
     protected function failedValidation(Validator $validator)
     {
         if ($this->isPrecognitive() || $this->wantsJson()) {
-            return (new ValidationException($validator))->errorBag($this->errorBag);
+            throw (new ValidationException($validator))->errorBag($this->errorBag);
```

Comment:
The reason I was using return rather than throw here and in the other requests is that it then gives the JSON response precognition expects rather than a standard response to a form (redirect in the failed test case). If you throw we'll need to build a response similar to how we do for `if ($this->ajax())`, but in the format Laravel expects:

```json
{
    "message": "The Email must be a valid email address.",
    "errors": {
        "email": [
            "The Email must be a valid email address."
        ]
    }
}
```

Reply:
Actually ignore that, I've found a different way to come at it. The issue was I wasn't actually sending a json request in the tests: https://github.com/ryanmitchell/cms/commit/04802e5391a24e99678dcb6962ead154b8cb77b5

Reply:
When it was `return`ing it just wasn't working at all. No precognition errors were being shown.

Reply:
Yep - my most recent push fixes that.

Reply:
Thanks!